packages/EasyUtils/src/Math.php

```diff
@@ -10,31 +10,24 @@
 
 final class Math implements MathInterface
 {
-    /**
-     * @var string
-     */
-    private $decimalSeparator;
+    private string $decimalSeparator;
 
-    /**
-     * @var int
-     */
-    private $roundMode;
+    private int $roundMode;
 
-    /**
-     * @var int
-     */
-    private $roundPrecision;
+    private int $roundPrecision;
 
-    /**
-     * @var int
-     */
-    private $scale;
+    private int $scale;
+
+    private string $thousandsSeparator;
 
     /**
-     * @var string
+     * @param int|null $roundMode [optional] <p>
```

Comment:
We've never used this kind of formatting before (with `<p>`). Is this something usual for the PhpDoc standard?

Reply:
I have copied this from https://github.com/JetBrains/phpstorm-stubs/blob/385f4627a8aae0d47b5496e87638354e3bdb30d4/standard/standard_3.php#L125

And I see it is legal - https://softwareengineering.stackexchange.com/questions/157283/html-in-docblock-comments